metals/src/main/scala/scala/meta/internal/metals/newScalaFile/NewFileTypes.scala

```diff
@@ -5,50 +5,83 @@ import scala.meta.internal.metals.MetalsQuickPickItem
 object NewFileTypes {
   sealed trait NewFileType {
     val id: String
+    val syntax: String
```

Comment:
```suggestion
    val syntax: Option[String]
```
to maybe avoid having `"NA"`